warehouse/core/src/main/java/datawave/util/TableName.java

```diff
@@ -0,0 +1,27 @@
+package datawave.util;
+
+/**
+ * Class that contains default table names
+ */
+public final class TableName {
+    
+    public static final String DATE_INDEX = "dateIndex";
```

Comment:
I've seen places in the code that are "DateIndex", should the first char be capitalized?

Reply:
From looking at all the instances of table names in the code, it appears that 'dateIndex' would be preferred over 'DateIndex'. I'm in favor of picking one standard convention